aibolit/patterns/instanceof/instance_of.py

```diff
@@ -15,16 +15,32 @@ def __traverse_node(self, node):
         List of code lines
         """
         lines = []
-        for _, node_elem in node.filter(javalang.tree.BinaryOperation):
-            if node_elem.operator == 'instanceof' and node_elem.operandl.position is not None:
-                code_line = node_elem.operandl.position.line or node_elem.operandr.position.line
-                lines.append(code_line)
-        for _, node_elem in node.filter(javalang.tree.MethodInvocation):
-            if node_elem.member == 'isInstance':
-                lines.append(node_elem.position.line)
-
+        depth = 0
+        cur_line = 0
+        bin_depth = -10
```

Comment:
What this magical numbers means?

Reply:
Initial depth for binary operation. You can use any number <= 0. Changed -10 to 0.